aws/resource_aws_ecs_service.go

```diff
@@ -379,6 +379,10 @@ func resourceAwsEcsService() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"wait_for_steady_state_timeout_in_minutes": {
```

Comment:
I'm not 100% sold on this property name and would love suggestions if anyone can come up with a better idea?

Another option I had was to put this in a element block like so,

```
wait_for_steady_state {
   enabled                     = true
   timeout_in_minutes = 15
}
```

Reply:
1. Block version is definitely more straightforward and easier to read
2. Most other timing-related parameters (health check timeout, deregistration delay, etc) are in seconds. Probably makes sense to follow that convention here also

Reply:
@brunhil Thank you for putting up this pr. Definitely a much needed addition. Could you please update so that it might get merged?

Reply:
I was hoping someone from the providers team would be able to provide some input on this potential config as well? 

Any chance you could provide input @YakDriver / @bflad / @anGie44 ?